File: Basics/BasicMaths/ReverseNumber/reverse.cpp
#include<iostream>
using namespace std;

int reverseNumber(int number){
    int x=number;
    int reverse=0;
    while(x){
        reverse=reverse*10+x%10;
        x=x/10;
    }
    return reverse;
}

int main(){
    int number=12345;
    cout<<"The reverse of "<<number<<" is "<<reverseNumber(number);
    return 0;
}

File: Basics/BasicMaths/ReverseNumber/reverse.js
const reverse = (num) => {
  let x = num;
  let number = 0;
  while (x) {
    number = number * 10 + (x % 10);
    x = Math.floor(x / 10);
  }
  return number;
};

let number = 12345;
console.log(`The reverse of number ${number} is ${reverse(number)}`);
